c/parallel/src/nvrtc/command_list.h

```diff
@@ -156,9 +156,23 @@ struct nvrtc_command_list_visitor
 
     check(jitlink_error);
 
-    check(nvJitLinkGetLinkedCubinSize(jitlink.handle, &cleanup.cubin_ref.size));
+    bool ptx    = false;
```

Comment:
For readability, maybe `output_kind_ptx` or similar?

Reply:
Changed in https://github.com/NVIDIA/cccl/pull/3462/commits/263fca32299cf70a1f3ab99feda97bddcce181c3.